bovheat_src/bh_input.py

```diff
@@ -0,0 +1,119 @@
+import os
+
+import pandas as pd
+
+
+def get_sourcedata(start_parameters, relative_path):
+
+    # Scan all file root and subfolders for xls and xslx files.
+    # Raise exception and exit if none are found.
+
+    try:
```

Comment:
Ich weiß du magst es nicht, aber das Error handling kann ruhig in der Main bleiben. Besonders da die Tests kein Enter drücken.

Reply:
read und get sourcedata sind auch sehr verwechselbar

Reply:
Yep. Finde ich keine ansehnliche Lösung. Aber reverted.